drv/stm32xx-sys/src/main.rs

```diff
@@ -398,7 +398,7 @@ fn main() -> ! {
                 // Process entries that are filled in...
                 if let &Some(ExtiDispatch { port, .. }) = entry {
                     let register = i >> 2;
-                    let slot = i % 2;
+                    let slot = i & 0b11;
```

Comment:
i waffled on whether `& 0b11` or `% 4` expresses intent more clearly --- i'm now mildly leaning towards `% 4` based on the observation that there are 4 bitfields per register. but, it doesn't actually matter.

I _do_ think that it might be nice if the `register` and `slot` calculation was below the comment describing what we're doing --- it's possible i didn't notice this in the review because i was too top-to-bottom in my reading habits, and didn't think to go back and look at the calculations after reading the comment explaining what they were doing. but, whether or not that would have prevented this is probably not really worth thinking too hard about :woman_shrugging: